user/hare/template.py

```diff
@@ -0,0 +1,42 @@
+pkgname = "hare"
+pkgver = "0.24.0"
+pkgrel = 0
+archs = ["aarch64", "riscv64", "x86_64"]
+build_style = "makefile"
+make_env = {"VERSION": pkgver, "LOCALVER": "chimera"}
+match self.profile().arch:
+    case "x86_64":
+        _qbe_arch = "amd64_sysv"
+    case "aarch64":
+        _qbe_arch = "arm64"
+    case "riscv64":
+        _qbe_arch = "rv64"
+    case _:
+        _qbe_arch = ""
+make_build_args = [
+    f"ARCH={self.profile().arch}",
+    f"QBEFLAGS=-t{_qbe_arch}",
+    f"{self.profile().arch.upper()}_CC=cc",
+    f"{self.profile().arch.upper()}_LD=ld",
+]
+# TODO: make it use clang assembler (at build time and also at runtime)
```

Comment:
@nekopsykose i vaguely remember you said you had done this already? not sure

Reply:
it doesn't work very well (forgot why), upstream requires gas basically so no need for the todo

Reply:
i mean sure, but still sucks to depend on it :/

i'll get rid of the todo for now but upstream told me they're open to making it work with clang so i'll look into it it at some point